paho/client.go

```diff
@@ -72,9 +72,8 @@ type (
 		Session          session.SessionManager
 		autoCloseSession bool
 
-		AuthHandler   Auther
-		PingHandler   Pinger
-		defaultPinger bool
```

Comment:
This should be kept and its value should be used to determine whether `SetDebugLogger` should call `SetDebug` on the `Pinger`.

Reply:
Thanks - I guessed it was there for something like that. Added code to set the logger.